src/sentry/logging/handlers.py

```diff
@@ -19,6 +19,18 @@ def emit(self, record):
         }
         if record.exc_info:
             kwargs['exc_info'] = record.exc_info
+
+        if '%' in record.msg and record.args:
+            try:
+                populated = record.msg % record.args
+            except TypeError:
```

Comment:
Two types, one is someone trying to pass a non-mapping to arguments which would come up with `TypeError: format requires a mapping` and specifying placeholders in the message that don't exist which would yield `TypeError: not all arguments converted during string formatting`. If it's worth raising them, we can remove this, but I wanted to make this as resilient as possible.